### PR TITLE
Update devices.js

### PR DESCRIPTION
**Pull request by tigercomp**
Support for Blaupunkt SSW-S1, clone of Bitron AV2010/34

### CODE REVIEW

**Comment by Koenkk** [comment]
Can you provide the data/database.db entry of this device?

**Comment by tigercomp** [comment]
{"id":10,"type":"EndDevice","ieeeAddr":"0x00124b0014b0a21c","nwkAddr":65238,"manufId":0,"manufName":"ClimaxTechnology","powerSource":"Battery","modelId":"WSS-4E_00.00.03.08TC","epList":[1],"endpoints":{"1":{"profId":260,"epId":1,"devId":4,"inClusterList":[0,1,3],"outClusterList":[3,4,5],"clusters":{"genBasic":{"attributes":{"modelId":"WSS-4E_00.00.03.08TC","manufacturerName":"ClimaxTechnology","powerSource":3,"zclVersion":1,"appVersion":6,"hwVersion":0,"dateCode":"20161007        "}}},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":6,"hwVersion":0,"dateCode":"20161007        ","zclVersion":1,"interviewCompleted":true,"meta":{},"lastSeen":1614423188560}

File: devices.js [hunk 1]
```diff
@@ -9140,6 +9140,20 @@ const devices = [
         },
         exposes: [e.cover_position().setAccess('state', ea.ALL)],
     },
+    {
+        zigbeeModel: ['SSW-S1'],
```
Comment (Koenkk) [inline devices.js:9144]: According to your database this should be `WSS-4E_00.00.03.08TC`. Does it really work with the current config?

Comment (tigercomp) [reply devices.js:9144]: Bad copy&paste, misplaced zigbeeModel and model.
`    {
        zigbeeModel: ['WSS-4E_00.00.03.08TC'],
        model: 'SSW-S1',
        vendor: 'Blaupunkt',
        description: '4-Touch single click buttons',
        fromZigbee: [fz.ignore_power_report, fz.command_recall, fz.legacy.AV2010_34_click],
        toZigbee: [],
        exposes: [e.action(['recall_*'])],
        meta: {configureKey: 1},
        configure: async (device, coordinatorEndpoint, logger) => {
            const endpoint = device.getEndpoint(1);
            await reporting.bind(endpoint, coordinatorEndpoint, ['genPowerCfg']);
        },
    },`

**Comment by Koenkk** [comment]
Updated this PR slightly, can you check if you also get a `battery` value now?

**Comment by tigercomp** [comment]
With external_converters this config won't load. My external_converter file:
```
const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
const tz = require('zigbee-herdsman-converters/converters/toZigbee');
const exposes = require('zigbee-herdsman-converters/lib/exposes');
const e = exposes.presets;

const device = {
        zigbeeModel: ['WSS-4E_00.00.03.08TC'],
        model: 'SSW-S1',
        vendor: 'Blaupunkt',
        description: '4-Touch single click buttons',
        fromZigbee: [fz.battery, fz.command_recall],
        toZigbee: [],
        exposes: [e.action(['recall_*']), e.battery()],
        meta: {configureKey: 1},
        configure: async (device, coordinatorEndpoint, logger) => {
            const endpoint = device.getEndpoint(1);
            await reporting.bind(endpoint, coordinatorEndpoint, ['genPowerCfg']);
        },
    };

module.exports = device;
```
And error on loading:
```
info  2021-02-28 14:12:12: SSW-S1 (0x00124b0014b0a21c): SSW-S1 - Blaupunkt 4-Touch single click buttons (EndDevice)
warn  2021-02-28 14:12:12: `permit_join` set to  `true` in configuration.yaml.
warn  2021-02-28 14:12:12: Allowing new devices to join.
warn  2021-02-28 14:12:12: Set `permit_join` to `false` once you joined all devices.
info  2021-02-28 14:12:12: Zigbee: allowing new devices to join.
info  2021-02-28 14:12:12: Configuring 'SSW-S1'
error 2021-02-28 14:12:12: Failed to configure 'SSW-S1', attempt 1 (ReferenceError: reporting is not defined
    at Object.configure (/app/lib/util/externally-loaded.js:16:13)
    at Configure.configure (/app/lib/extension/configure.js:131:45)
    at Configure.onZigbeeStarted (/app/lib/extension/configure.js:99:28)
    at Controller.callExtensionMethod (/app/lib/controller.js:380:44))
```

**Comment by sjorge** [comment]
I think you're missing 

```
const reporting = require('zigbee-herdsman-converters/lib/reporting');
```

**Comment by szakals** [comment]
[addon_45df7312_zigbee2mqtt_logs.txt](https://github.com/Koenkk/zigbee-herdsman-converters/files/6078890/addon_45df7312_zigbee2mqtt_logs.txt)

In the attachment, the pairing log for zigbee_herdsman_debug: true
config
const fz = {...require('/config/zigbee2mqtt/converters/fromZigbee'), legacy: require('/config/zigbee2mqtt/lib/legacy').fromZigbee};
const tz = require('zigbee-herdsman-converters/converters/toZigbee');
const exposes = require('/config/zigbee2mqtt/lib/exposes');
const reporting = require('/config/zigbee2mqtt/lib/reporting');
const e = exposes.presets;

const device = {
        zigbeeModel: ['WSS-4E_00.00.03.08TC'],
        model: 'SSW-S1',
        vendor: 'Blaupunkt',
        description: '4-Touch single click buttons',
        fromZigbee: [fz.battery, fz.command_recall],
        toZigbee: [],
        exposes: [e.action(['recall_*']), e.battery()],
        meta: {configureKey: 1},
        configure: async (device, coordinatorEndpoint, logger) => {
            const endpoint = device.getEndpoint(1);
            await reporting.bind(endpoint, coordinatorEndpoint, ['genPowerCfg']);
        },
    };

module.exports = device;

**Pull request closed without merging**